python/src/ir.cc

```diff
@@ -1780,9 +1784,8 @@ void init_triton_ir(py::module &&m) {
              }
              if (haveDump) {
                context->disableMultithreading();
-               auto printingFlags = OpPrintingFlags();
+               auto printingFlags = getOpPrintingFlags();
                printingFlags.elideLargeElementsAttrs(16);
```

Comment:
In theory, we should either set this at all places or just take it out. I'm not sure what to do here, so just leave it for backward compatibility

Reply:
I think this flag makes the printed IR incomplete, as it will print `dense<__elided__>` for large constants. Perhaps we should maybe remove it entirely.

Reply:
Tracing back the history I found it was added during the initial MLIR development. Let's just take it out anyway